### DISCUSSION TITLE
Welcome to SORMAS-Glossary Discussions!

### DISCUSSION THREAD

**Discussion by JonasCir** [hzi-braunschweig/SORMAS-Glossary#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JonasCir** [comment]
@Candice-Louw @bernardsilenou 
Github discussions is [here](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-team-discussions)

:smiley:

**Comment by bernardsilenou** [comment]
Nice one J,
Lets give it a try.
Thanks

**Comment by bernardsilenou** [comment]
@BasilKaburi hope you got the invite?

**Reply by JonasCir** [reply]
It's open to everyone! Public access :)
